test/e2e/aks_public_ip_prefix.go

```diff
@@ -82,7 +82,7 @@ func AKSPublicIPPrefixSpec(ctx context.Context, inputGetter func() AKSPublicIPPr
 		g.Expect(err).NotTo(HaveOccurred())
 	}, input.WaitIntervals...).Should(Succeed(), "failed to create public IP prefix")
 
-	By("Creating node pool with 3 nodes")
+	By("Creating node pool with 2 nodes")
```

Comment:
you need to also change the replica count on line 108 for this to be true :)

Reply:
Corrigé, merci !